package.json

```diff
@@ -52,8 +88,14 @@
         "husky": "^7.0.4",
         "lint-staged": "^12.1.2",
```

Comment:
lint-staged may require additional configuration in `package.json` to use cache.
This section is added in top level `package.json` may be needed to update with TS extensions?
```json
"lint-staged": {
    "*..{js,ts,tsx}": "eslint --cache --fix"
  },
```